.github/scripts/build_publish_nightly_docker.sh

```diff
@@ -3,7 +3,7 @@
 set -xeuo pipefail
 
 PYTORCH_DOCKER_TAG=$(git describe --tags --always)-devel
-CUDA_VERSION=11.3.1
+CUDA_VERSION=11.6.1
```

Comment:
Is there a reason why you want 11.6.1 version in particular
```suggestion
CUDA_VERSION=11.6.2
```

Reply:
No particular reason I think 11.6.2 should work just as fine